src/compas_ghpython/uninstall.py

```diff
@@ -14,7 +14,7 @@ def after_rhino_uninstall(uninstalled_packages):
     if 'compas_ghpython' not in uninstalled_packages:
         return []
 
-    srcdir = os.path.join(os.path.dirname(__file__), 'components')
+    srcdir = os.path.join(os.path.dirname(__file__), 'components', 'ghuser')
```

Comment:
It does not fail for COMPAS core because of the "fallback" mechanism in lines 100-101 of `compas_ghpython.components.uninstall_userobjects()`, which searches for .ghuser components whose names start with "compas":

```python
# compas_ghpyyhon.components.uninstall_userobjects(user_objects)

userobjects = []   # at the moment, this silently overwrites the userobjects in the function's signature
for name in os.listdir(dstdir):
      if name.lower().startswith('compas'):
      userobjects.append(name)
```

The fallback makes sense because `userobjects` below is currently an empty list. Why? Because the core .ghuser components are installed by `compas_ghpython.install_userobjects` in `os.path.dirname(__file__)/components/ghuser` not in `os.path.dirname(__file__)/components`.
If you install the COMPAS core GH components --either from a source or from a conda installation--, and then uninstall them, the following check fails.

```python
# compas_ghpython.uninstall.py
srcdir = os.path.join(os.path.dirname(__file__), 'components')
userobjects = [os.path.basename(ghuser) for ghuser in glob.glob(os.path.join(srcdir, '*.ghuser'))]
assert len(userobjects) > 0   # it assumes ghuser components were first installed 
uninstalled_objects = uninstall_userobjects(userobjects)
```

If we change the path for uninstall as proposed in this PR, the check passes, and the COMPAS core and extensions packages ghuser components are uninstalled without breaking.

Reply:
What do you think, @gonzalocasas? Reasonable or should I roll back this one line to help merging this PR? 🙂

Reply:
think he is busy with a workshop...

Reply:
@brgcode Thanks for the heads up!